test/controllers/bill-runs.controller.test.js

```diff
@@ -101,4 +101,23 @@ describe('Bill Runs controller', () => {
       })
     })
   })
+
+  describe('GET /bill-runs/{id}/review', () => {
+    let options
+
+    beforeEach(async () => {
+      options = {
+        method: 'POST',
+        url: '/charge-elements/time-limited'
+      }
+    })
+
+    describe('when a request is valid', () => {
+      it('returns a 200 response', async () => {
+        const response = await server.inject(options)
+
+        expect(response.statusCode).to.equal(204)
```

Comment:
And if you correct the URL I suggested then I would expect this to break unless it gets changed to ...

```suggestion
        expect(response.statusCode).to.equal(200)
```